docs/en/drivers/ios-xcuitest-real-devices.md

```diff
@@ -130,7 +130,7 @@ installed, and is _not_ recommended):
 ```
 *   Given this installation location, `/path/where/installed/bin/appium`, `WebDriverAgent` project
     will be found in `/path/where/installed/lib/node_modules/appium/node_modules/appium-webdriveragent`.
-    Open a terminal and go to that location, then run the following in order to
+*  (Only for lower than Appium 1.20) Open a terminal and go to that location, then run the following in order to
```

Comment:
Only for Appium versions below 1.20

Reply:
Also we still need to create the bundle folder, only the second step should be skipped